plugins/outputs/cloudwatch/README.md

```diff
@@ -36,3 +36,13 @@ Examples include but are not limited to:
 ### namespace
 
 The namespace used for AWS CloudWatch metrics.
+
+### write_statistics
+
+If you have a large amount of metrics, you should consider to send statistic 
```

Comment:
Some minor grammar:
 - s/consider to send/consider sending/
 - s/ which could/. This should/
 - s/If enable this flag/If this flag is enabled/